source/patterns/01-molecules/info-bar/info-bar.twig

```diff
@@ -0,0 +1,38 @@
+{%- import 'atoms-html' as html -%}
+
+{%- set imagePath = {
+  attention: asset('images/attention.svg'),
+  correction: asset('images/corrected-article.svg'),
+  info: asset('images/info.svg'),
+  success: asset('images/success.svg'),
+  warning: null,
```

Comment:
Redundant to call `asset()` on all cases. Thinking though it's probably better to decouple the image from the colour (can lose `correction` as a type). So the path would be input. Can add custom images if needed easily too then.

Reply:
Refactored to inject image path 497b3dd.